internal/modifier/graphics_test.go

```diff
@@ -92,6 +92,7 @@ func TestGraphicsModifier(t *testing.T) {
 		t.Run(tc.description, func(t *testing.T) {
 			image, _ := image.New(
 				image.WithEnvMap(tc.envmap),
+				image.WithAcceptEnvvarUnprivileged(true),
```

Comment:
One thing that this calls out is that the *default* for `acceptEnvvarUnprivileged` is `true`, so maybe one should capture that in the image itself?

Reply:
not on this PR, I feel that could be an API change. don't you think? (I agree with the change)

Reply:
That's not what I mean. I mean the default value for the member in the struct should be true.